packages/quantic/quantic-examples-community/objects/Case.object

```diff
@@ -0,0 +1,421 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we should have that either, it's for giving access to the example community right?

Did you try with a permission set instead?

Reply:
What is the line exactly that grants the permissions? I see the permission on the profile you have included already, don't think we need the object on top of the profile

Reply:
True! I just retried to setup the community after removing the objects and the community was created correctly with the setting we needed. I will just double check cause I remember trying this previously and it didn't give me the needed results without including the objects. when I confirm that it's working correctly I will just delete the files related to the objects and I'm gonna keep only the profile metadata👌